From 9160a127b5069ee1ca3bdf11857a04f75099041d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=89rico=20Nogueira?= <erico.erc@gmail.com>
Date: Tue, 20 Jul 2021 12:01:21 -0300
Subject: [PATCH] modules: fix typo in module-rtkit

Was using the gettid() function directly instead of the _gettid()
wrapper.

[Retrieved from:
https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/9160a127b5069ee1ca3bdf11857a04f75099041d]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 src/modules/module-rtkit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/module-rtkit.c b/src/modules/module-rtkit.c
index de30f4123..419e4be81 100644
--- a/src/modules/module-rtkit.c
+++ b/src/modules/module-rtkit.c
@@ -518,7 +518,7 @@ static void *custom_start(void *data)
 	struct impl *impl = this->impl;
 
 	pthread_mutex_lock(&impl->lock);
-	this->pid = gettid();
+	this->pid = _gettid();
 	pthread_cond_broadcast(&impl->cond);
 	pthread_mutex_unlock(&impl->lock);
 
-- 
GitLab

